packages/dev-tools/package.json

```diff
@@ -54,6 +54,7 @@
     "copy-to-clipboard": "^3.0.8",
     "emotion": "^8.0.5",
     "emotion-server": "^8.0.5",
+    "expo-optimize": "0.0.0",
```

Comment:
I don't think `expo-optimize` should be a dependency here.

Reply:
Will this still be able to resolve if we remove it? https://github.com/expo/expo-cli/pull/1192/files#diff-d087e14e7de95f68c0b8fac69087f279R571

Reply:
No it won't work unless it's a dependency. Could we spawn `npx` instead, or (maybe better) use `libnpx` (the programmatic API to npx) to run it?

Reply:
libnpx seems deprecated https://github.com/zkat/npx

Reply:
just using whichever globally installed npx module exists directly for now